### PR TITLE
fake: Add version 5.18.3

### PR DESCRIPTION
**Pull request by kostrse**
FAKE package manager
https://fake.build/

Similar to Cake:
https://cakebuild.net/

### CODE REVIEW

**Comment by Ash258** [comment]
What is difference between `fake-dotnetcore-portable.zip` and `fake-dotnetcore-win7-x64.zip`?

**Comment by kostrse** [comment]
The portable package doesn't contain .NET Core runtime, only platform independent application binaries. It requires .NET Core runtime to run, it doesn't have own .exe and needs dotnet to be executed. The second package is self-containing, has .NET runtime packaged and own .exe file, has no external dependencies.

**Comment by kostrse** [comment]
Hi @matthid, I would want to publish FAKE to the scoop package manager.

If you don't mind, could you please have a quick look and give feedback whether it aligns well with intended usage scenarios of FAKE 5?

Did we pick correct binaries?

**Comment by matthid** [comment]
Sure, sounds reasonable to me. I don't know how "updating" works with Scoop, but if there is anything we can/should automate feel free to extend the release-pipeline in fake (or open an issue in case any help from my side is needed).

> Did we pick correct binaries?

I can not really answer this because I don't know how Scoop works, but I guess people expect to have a working `fake`-Command in their shell after installing. The easiest way to achieve this is indeed `fake-dotnetcore-win7-x64.zip` (just extract and run basically), but it is also the most expensive download. If scoop can install wrapper scripts and depend on other packages the download-times could be improved by using the portable one and depending on dotnet-sdk.

File: bucket/fake.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://fake.build/",
+    "description": "FAKE - F# Make, is a build automation system with capabilities which are similar to make and rake.",
```
[the remark this answers is not in the document]
Comment (kostrse) [reply bucket/fake.json]: I bit changed the wording, keep mention of F# and that this is a build tool for .NET

**Pull request merged**